src/Shared/AppModels/AppRoot.cs

```diff
@@ -373,6 +389,15 @@ private async Task HandleCoreSettingsCollectionChangedAsync<TSettings>(object? s
                         Logger.LogInformation($"Adding new core {newCore.DisplayName}, {nameof(newCore.InstanceId)} {newCore.InstanceId}");
                         _mergedCore.AddSource(newCore);
                     }
+
+                    Guard.IsNotNull(Ipfs);
```

Comment:
This line needs to be removed. Ipfs is optional in Strix, this will crash if the user doesn't have IPFS enabled.